public/index.html

```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
```

Comment:
Est-ce que tu pourras définir la font comme paramètre du widget stp ? Comme ça, tu n'auras pas à l'inclure dans ton code et les utilisateurs de ton widget pourront la personnaliser.

Reply:
La balise link restera bien dans le fichier index.html. Par contre, tu peux ajouter dans le state de ton component une propriété "font" qui sera définie dans le constructor : this.state= { font: this.props.font }